app/assets/stylesheets/invitations.css.scss

```diff
@@ -5,11 +5,19 @@
 }
```

Comment:
a little bit up here, in you should set the heigth of the `#invite_code` field
`height: 1.6rem;`
worked fine for me, it's large enough that you can see the `p` descendant in `http` from the invite.
It's cool to user rems, because if you increase the font-size on the `html` or `body` element, the height will grow proportionally. rems, ems and percentages are responsive.